### PR TITLE
Helm charts to deploy SAM decoder and encoder

### PR DESCRIPTION
**Pull request by Rub21**
here are some helm charts to deploy SAM decoder and encoder in a kubernetes cluster

cc. @batpad @yunica @ingalls  @rbavery

### CODE REVIEW

File: sam-helm-charts/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: sam
+description: A Helm chart for Kubernetes
```
Comment (batpad) [inline sam-helm-charts/Chart.yaml:3]: Can you add a short description of what this Helm chart helps you to install here?

File: sam-helm-charts/values.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# encoder
+# ====================================================
+encoder:
+  enabled: true
+  replicas: 1
+  serviceType: LoadBalancer
+  serviceAnnotations:
+    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "300"
+  image:
+    name: 685980093638.dkr.ecr.us-east-1.amazonaws.com/sam-service
```
Comment (Rub21) [inline sam-helm-charts/values.yaml:16]: @batpad , currently we  don't have a github a action for this deployment, i ma just adding this values  form manual deployment.

Comment (batpad) [reply sam-helm-charts/values.yaml:16]: Yea, I don't think any of these values are sensitive and I think this is fine for now.

**Review by batpad — APPROVED** [review]

**Comment by batpad** [comment]
@Rub21 apologies I missed coming back to this - we should merge this!

**Pull request merged**